ngrinder-controller/src/main/resources/messages_en.properties

```diff
@@ -207,6 +207,7 @@ perfTest.config.process=Processes
 perfTest.config.thread=Threads
 perfTest.config.ignoreTooManyError=Ignore Errors
 perfTest.config.ignoreTooManyError.help=If you want to continue testing even if too many errors occur. Please check this.
+perfTest.config.connectionReset=Connection reset after one test cycle
```

Comment:
Connection reset on each test run.

Reply:
The reason I used the word `test cycle` is `each test` might be confusing in meaning like below.

1. Is it reset on each `@Test` test function?
2. Is it reset on each test cycle?

but, `Connection reset on each test run.` seems to provide a better understanding to users.

Reply:
"run" is official word in grinder world. See.. "run count"

Reply:
The reason why I recommend "Connection reset on each test run" rather than"Connection reset after one test cycle", it's little bit shorter than what you used.